File: forloop-add-test/forloop-add-test.js
#!/usr/bin/env nodejs

var toval = parseInt( process.argv.slice(2) );
var i, j, sumval = 0;
for (i=0; i<toval; i++) {
    for (j=0; j<toval; j++) {
        sumval += i+(j^1);
    }
}

console.log("nodejs:", sumval);

File: forloop-add-test/forloop_add_test.lua
#!/usr/bin/env luajit

local toval = tonumber(arg[1])
local sumval = 0
for i = 0, toval-1 do
    for j = 0, toval-1 do
        sumval = sumval + i+(j^1)
    end
end
print("Luajit:", sumval);


